go.mod

```diff
@@ -6,17 +6,16 @@ require (
 	github.com/Stebalien/go-bitfield v0.0.1
 	github.com/ipfs/go-bitfield v1.0.0
 	github.com/ipfs/go-cid v0.1.0
-	github.com/ipfs/go-datastore v0.4.6 // indirect
 	github.com/ipfs/go-ipfs-chunker v0.0.1
 	github.com/ipfs/go-ipfs-util v0.0.2
 	github.com/ipfs/go-ipld-format v0.2.0
-	github.com/ipfs/go-merkledag v0.3.2
+	github.com/ipfs/go-merkledag v0.5.1
 	github.com/ipfs/go-unixfs v0.2.4
-	github.com/ipld/go-car/v2 v2.1.0
+	github.com/ipld/go-car/v2 v2.1.1
 	github.com/ipld/go-codec-dagpb v1.3.0
-	github.com/ipld/go-ipld-prime v0.12.4-0.20211014180653-3ba656a3bc6b
-	github.com/multiformats/go-multicodec v0.3.1-0.20210902112759-1539a079fd61
-	github.com/multiformats/go-multihash v0.0.17-0.20211012170226-654b06d7912a
+	github.com/ipld/go-ipld-prime v0.14.3
+	github.com/multiformats/go-multicodec v0.3.1-0.20211210143421-a526f306ed2c
```

Comment:
Seems like we're waiting on a release of go-multicodec. Anything we're waiting on @mvdan?

Reply:
https://github.com/multiformats/go-multicodec/releases/tag/v0.4.0

Reply:
^ bump in case you didn't see it during the holidays :)